src/wallet/wallet_rpc_server.h

```diff
@@ -110,6 +110,14 @@ namespace tools
         MAP_JON_RPC_WE("get_languages",      on_get_languages,      wallet_rpc::COMMAND_RPC_GET_LANGUAGES)
         MAP_JON_RPC_WE("create_wallet",      on_create_wallet,      wallet_rpc::COMMAND_RPC_CREATE_WALLET)
         MAP_JON_RPC_WE("open_wallet",        on_open_wallet,        wallet_rpc::COMMAND_RPC_OPEN_WALLET)
+        MAP_JON_RPC_WE("is_multisig",        on_is_multisig,        wallet_rpc::COMMAND_RPC_IS_MULTISIG)
+        MAP_JON_RPC_WE("get_multisig_info",  on_get_multisig_info,  wallet_rpc::COMMAND_RPC_GET_MULTISIG_INFO)
```

Comment:
I think the command names should be made consistent with the CLI to avoid unnecessary confusion.